notebooks/tfx_pipelines/guided_projects/tfx_guided_project_vertex.ipynb

```diff
@@ -0,0 +1,753 @@
+{
```

Comment:
<p>Suggestion: Add a note that the notebook kernel might need to be interrupted to get the next cells to run. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/157/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>

Reply:
<p>Done!</p>